content-learn/mastering-service-meshes-for-operators/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: 'Mastering Service Meshes for Operators'
+description: 'Learn how to deploy a service mesh manage its workloads'
```

Comment:
nit: service mesh and manage it's workload

Reply:
done